packages/components/src/pages/ChatSearch.vue

```diff
@@ -39,38 +40,89 @@
       data-cy="resize-handle"
       @mousedown="startResizing"
     ></div>
-    <div class="search-container">
-      <v-context
-        class="no-match-instructions"
-        :appmap-stats="appmapStats"
-        :context-response="contextResponse"
-        :pinned-items="pinnedItems"
-      />
+    <div class="search-wrapper">
+      <div class="search-header">
+        <h1>Navie's Context Sources</h1>
+        <!-- <div class="info"  @click.prevent="onMouseEnter" @mouseenter="onMouseEnter" @mouseleave="onMouseLeave"> -->
+        <v-popper class="infoHelp" ref="infoPopper" placement="none">
+          <div class="info" @click.prevent="onMouseEnter">
+            <InfoIcon />
+          </div>
+          <template #content>
+            <p>The Context Window displays data sources that Navie is using to inform responses.</p>
+            <p>
+              <strong>Pinned Items</strong> - specific content that you select to refine AI
+              responses.
+            </p>
+            <p>
+              <strong>Selected by Navie</strong> - code, files, and data relevant to the issue you
+              are working on that Navie has discovered in your environment. You can give Navie more
+              to work with by creating AppMap data specific to code you are working on.
+            </p>
+          </template>
+        </v-popper>
+      </div>
+      <div class="pinned-items-wrapper">
+        <div class="search-subheader">
+          <h2>
+            Pinned Items
+            <span class="pinned-item__count">{{ pinnedItemCount }}</span>
+          </h2>
+          <v-add-file-button
+            v-if="hasPinnedItems || showAddFilesWhenEmpty"
+            @click.native="addFiles"
+          />
+        </div>
+        <div class="search-container">
+          <v-pinned-items
+            class="no-match-instructions"
+            :pinned-items="pinnedItems"
+            :editor-type="editorType"
+          />
+        </div>
+      </div>
+      <div class="context-wrapper">
+        <h2>Selected by Navie</h2>
+        <div class="search-container">
+          <v-context
+            class="no-match-instructions"
+            :appmap-stats="appmapStats"
+            :context-response="contextResponse"
+          />
+        </div>
+      </div>
```

Comment:
These aren't scrolling the correct area. The scroll bars should span the entire height of the container. The container should also not scroll unless the content is overflowing.
![image](https://github.com/user-attachments/assets/96005f5a-4287-4e9d-8a45-9d084f7f12fa)

Reply:
Notice the placement and sizing of the scrollbars in these two images. The container would ideally take up its maximum height even before filling up with context items.

![image](https://github.com/user-attachments/assets/abdad12e-75d3-4e60-9d0a-8c7ec5c0d57d)

![image](https://github.com/user-attachments/assets/11f0bb75-c898-4b16-8039-d0955069447c)

Reply:
Yeah, that's the problem I fixed. I thought you were saying there was an additional problem, but maybe not?

Reply:
It looks better now